.env

```diff
@@ -1,6 +1,6 @@
 # Rename this file to .env and chang the values accordingly [refer to docs/120823-neon-tech-db-info.md]
 DB_USER=sally
-DB_HOST=ep-morning-dawn-18951933
+DB_HOST=ep-cold-poetry-404091.us-east-2.aws.neon.tech
 DB_NAME=neondb
-DB_PASSWORD=PZRStJ9bmF5A
```

Comment:
I apologise for not clarifying. The "Testing" steps are meant to be run locally only.

`.env` files usually contain secrets, keys to paid APIs or credentials to confidential information. We should try to avoid sharing and pushing these values to GitHub. 

In this case, by pushing the password and db information to GitHub, anyone can access and modify the Database. In a real environment, we have just leaked all our user's personal information.

**Suggestions:**
- Look into `git rebase`
- Drop this latest commit from history
- Push with `--force` or `--force-with-lease`
- Make a commit, adding `.env` to `.gitignore` to prevent this from happening in the future
- This repository is public, there are bots that probably scraped this PR and saved the DB details. Consider deleting the DB you currently have and creating a new one.